Add doc about custom http timeout in RabbitMQ Scaler (#523)
kedacore/keda-docs: content/docs/2.5/scalers/rabbitmq-queue.md

- `queueLength`: DEPRECATED! Use `mode: QueueLength` and `value: ##` instead. Target value for queue length passed to the scaler. Example: if one pod can handle 10 messages, set the queue length target to 10. If the actual number of messages in the queue is 30, the scaler scales to 3 pods. Default is 20 unless `publishRate` is specified, in which case `queueLength` is disabled for this trigger.
- `useRegex`: In case of `http` protocol, this parameter allows to use regex (in `queueName` parameter) to select queue instead of full name, the valid values are: `"true"` and `"false"`.  Optional.
- `operation`: Operation that will be applied to compute the number of messages in case of `useRegex` enabled. Either `sum` (default),`max`, or `avg`. Optional.
- `metricName`: an optional name to assign to the metric. If not set KEDA will generate a name based on the queue name. If using more than one trigger it is required that all metricNames be unique.

> - `timeout`: Timeout **for the specific trigger** in case of `http` protocol. This value will override the value defined in `KEDA_HTTP_DEFAULT_TIMEOUT`. Optional.

Reviewer: What do you mean with "for the specific trigger"?

Author: That this timeout only for this trigger, and it will not affect to other triggers. You can have 2 different triggers that uses RabbitMQ with totally different timeouts. (Maybe is my translation a "false friend"?)

Reviewer 2: Maybe use `for this specific trigger`?

Reviewer: That, or change the docs for the "global" setting to emphasize that it's global because the trigger docs typically focus on the trigger itself.

If you know what I mean 😅

Author: I got it, I can do both :)
I have already updated to change from for the specific trigger to for this specific trigger, but I can change also this "This value will override the value defined in KEDA_HTTP_DEFAULT_TIMEOUT" with "This value will override the global value defined in KEDA_HTTP_DEFAULT_TIMEOUT"
Is it better in your opinion?

Reviewer: Where is KEDA_HTTP_DEFAULT_TIMEOUT documented other than the Helm chart? I'd make it explicit there that this is global

Author: It's already explained [in docs](https://keda.sh/docs/2.1/operate/cluster/#http-timeouts)
> Some scalers issue HTTP requests to external servers (i.e. cloud services). Each applicable scaler uses its own dedicated HTTP client with its own connection pool, and by default each client is set to time out any HTTP request after 3 seconds.
 ...

Reviewer: Good enough, thanks!

Author: okey, then I leave it as it is now :)

> - `timeout`: Timeout **for this specific trigger** in case of `http` protocol. This value will override the value defined in `KEDA_HTTP_DEFAULT_TIMEOUT`. Optional.

Reviewer 3: @JorTurFer does the first sentence mean to say that the timeout only applies for this specific trigger, only if it's the `http` protocol? If so, here's a suggestion:

```suggestion
- `timeout`: Timeout **for this specific trigger**. This value will override the value defined in `KEDA_HTTP_DEFAULT_TIMEOUT`. Optional. Only apples to hosts that use the `http` protocol
```

Author: Nice suggestion! Thanks for do it :)

Reviewer 3: No problem :)
